src/Compilers/CSharp/Test/Emit3/Attributes/AttributeTests.cs

```diff
@@ -2396,6 +2396,9 @@ public class Test
                 // (6,21): warning CS0657: 'field' is not a valid attribute location for this declaration. Valid attribute locations for this declaration are 'method, return'. All attributes in this block will be ignored.
                 //     public int P { [field: A] get => throw null; set => throw null; }
                 Diagnostic(ErrorCode.WRN_AttributeLocationOnBadDeclaration, "field").WithArguments("field", "method, return").WithLocation(6, 21),
+                // (6,50): warning CS9265: The 'set' accessor of property 'Test.P' should use the backing 'field' because the other accessor is using it.
+                //     public int P { [field: A] get => throw null; set => throw null; }
```

Comment:
>public int P { [field: A] get => throw null; set => throw null; } [](http://example.com/codeflow?start=23&length=65)

Are we generating a backing field for this property? That seems like a bug, even though the attribute is an error.

Reply:
Logged https://github.com/dotnet/roslyn/issues/75459.

Reply:
This issue should be resolved by https://github.com/dotnet/roslyn/pull/75461.